website/docs/reference/resource-configs/teradata-configs.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Teradata configurations"
+id: "teradata-configs"
+---
+
+## General
+
+* *Set `quote_columns`* - to prevent a warning, make sure to explicitly set a value for `quote_columns` in your `dbt_project.yml`. See the [doc on quote_columns](https://docs.getdbt.com/reference/resource-configs/quote_columns) for more information.
+
+```yaml
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
+```
+
+* *Enable view column types in docs* - Teradata Vantage has a dbscontrol configuration flag called `DisableQVCI`. This flag instructs the database to create `DBC.ColumnsJQV` with view column type definitions. To enable this functionality you need to:
+  1. Enable QVCI mode in Vantage. Use `dbscontrol` utility and then restart Teradata. Run these commands as a privileged user on a Teradata node:
+      ```bash
+      # option 551 is DisableQVCI. Setting it to false enables QVCI.
+      dbscontrol << EOF
+      M internal 551=false
+      W
+      EOF
+
+      # restart Teradata
+      tpareset -y Enable QVCI
+      ```
+  2. Instruct `dbt` to use `QVCI` mode. Include the following variable in your `dbt_project.yml`:
```

Comment:
Unfortunately, I can't seem to get to the deploy working correctly but I think you might need a space before line 17 and before line 27.

Reply:
@amychen1776 , thank you for reviewing the PR. I've checked locally and this fragment renders OK:

![image](https://user-images.githubusercontent.com/6579240/149971884-56296185-be9e-4512-b628-eace1e5b2982.png)

Reply:
Ah epic! Thank you for the confirmation. This is nitpicky but could you alias the link URL?

Reply:
thanks for the suggestion. Done.